de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/location/LocationId.java

```diff
@@ -1,5 +1,7 @@
 package org.adempiere.location;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
@TheBestPessimist @teosarca wdyt, maybe we should now move all this stuff we touch from from `org.adempiere.location` to `de.metas.location` wdyt, how much effort would that be?

Reply:
imho, minor. aggree, let's do it!

Reply:
done